panda/src/egg/eggLine.h

```diff
@@ -31,15 +31,15 @@ class EXPCL_PANDAEGG EggLine : public EggCompositePrimitive {
 
   virtual EggLine *make_copy() const OVERRIDE;
 
-  virtual void write(ostream &out, int indent_level) const;
+  virtual void write(ostream &out, int indent_level) const OVERRIDE;
```

Comment:
You can feel free to use lowercase `override` since all the compilers we target nowadays support it.